src/UIState/SeePIDConstants.cpp

```diff
@@ -4,12 +4,68 @@
 
 #include "SeePIDConstants.h"
 
-void SeePIDConstants::handleKey(char key) {
-  switch (key) {
-    case 'D':  // Don't save (cancel)
-      returnToMainMenu();
+#include "Devices/LiquidCrystal_TC.h"
+#include "Devices/PHProbe.h"
+#include "Devices/PID_TC.h"
+#include "Serial_TC.h"
+#include "TC_util.h"
+
+void SeePIDConstants::loop() {
+  int elapsedSeconds = (millis() - startTime) / 1000;
+
+  Serial_TC *mySerial = Serial_TC::instance();
+  mySerial->print("SeePIDConstants::loop() - 1 CurrentTime ", elapsedSeconds);
```

Comment:
If this was for debugging, it should come out. I don't think we want to log a new line every time through the loop.